polkadot/zombienet-sdk-tests/tests/parachains/warm_up_trie_cache.rs

```diff
@@ -0,0 +1,334 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// TODO: Use precompiled metadata when v15 has become by default
+// `#[subxt::subxt(runtime_metadata_path = "metadata-files/asset-hub-westend-local.scale")]`
+// Don't forget to remove `subxt-macro` from all Cargo.toml files
+#[subxt::subxt(runtime_metadata_insecure_url = "wss://westend-asset-hub-rpc.polkadot.io:443")]
+mod ahw {}
+
+use ahw::runtime_types::{
+	pallet_revive::primitives::{Code, StorageDeposit},
+	sp_weights::weight_v2::Weight,
+};
+use anyhow::anyhow;
+use asset_hub_westend_runtime::Runtime as AHWRuntime;
+use futures::{stream::FuturesUnordered, StreamExt};
+use pallet_revive::AddressMapper;
+use sp_core::H160;
+use std::str::FromStr;
+use subxt::{
+	config::polkadot::PolkadotExtrinsicParamsBuilder, tx::SubmittableExtrinsic, OnlineClient,
+	PolkadotConfig,
+};
+use subxt_signer::{
+	sr25519::{dev, Keypair},
+	SecretUri,
+};
+use zombienet_sdk::{LocalFileSystem, Network, NetworkConfigBuilder};
+
+/// Verifies the effectiveness of the trie cache warming mechanism
+///
+/// Passes if the cache hit rate is above 85%, indicating that the trie cache
+/// was successfully warmed up and is effectively reducing storage access after node restart.
```

Comment:
This will always fail on CI or will be ultra flakely. I don't really see the benefit. Especially as you are starting entire nodes. All this could just be some benchmark or similar.

Reply:
Launching a network to test this, sounds like an overkill to me.